src/rprblender/properties/object.py

```diff
@@ -73,6 +73,11 @@ class RPR_ObjectProperites(RPR_Properties):
         description="Use this object as a portal light",
         default=False,
     )
+    visibility_contour: BoolProperty(
```

Comment:
What would be the reason you'd want an object to be invisible in contour mode?

Reply:
It is the part of Parameters and UI sections of Contour.pptx attached to the task

Reply:
Ok.  I'll talk to SDK team.  This is a weird setting it should just use camera visibility.